app/src/main/java/org/mozilla/focus/fragment/CrashReporterFragment.kt

```diff
@@ -5,35 +5,27 @@
 package org.mozilla.focus.fragment
 
 import android.os.Bundle
-import android.view.LayoutInflater
 import android.view.View
-import android.view.ViewGroup
 import androidx.appcompat.content.res.AppCompatResources
 import androidx.fragment.app.Fragment
-import kotlinx.android.synthetic.main.fragment_crash_reporter.*
 import mozilla.components.service.glean.private.NoExtras
 import org.mozilla.focus.GleanMetrics.CrashReporter
 import org.mozilla.focus.R
+import org.mozilla.focus.databinding.FragmentCrashReporterBinding
 
 class CrashReporterFragment : Fragment() {
     var onCloseTabPressed: ((sendCrashReport: Boolean) -> Unit)? = null
 
-    override fun onCreateView(
-        inflater: LayoutInflater,
-        container: ViewGroup?,
-        savedInstanceState: Bundle?
-    ): View? = inflater.inflate(R.layout.fragment_crash_reporter, container, false)
-
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-
-        view.findViewById<View>(R.id.background).background =
+        val binding = FragmentCrashReporterBinding.bind(view)
```

Comment:
We are trying to bind here a View which wasn't previously created - inflated.
As such this method - `onViewCreated` won't even be called.

Nice that you've removed the almost empty `onCreateView` but if not inflating the layout from there we should use the Fragment constructor taking the layout resource as a parameter like in 
```
class CrashReporterFragment : Fragment(R.layout.fragment_crash_reporter) { .. }
```

This fragment's functionality can then be tested using `about:crashtab`.